modules/ROOT/pages/runtime-patch-updates.adoc

```diff
@@ -93,10 +92,12 @@ include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[
 
 include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=MuleIntro6]
 
+After you select the option *Use Edge Release Channel* in the Runtime Manager UI, the platform shows all the build tags for Edge and LTS channels. The build tag is specific to Runtime Manager and CloudHub 2 because the image can be built multiple times for a given Mule runtime version. 
```

Comment:
```suggestion
After you select the option *Use Edge Release Channel* in the Runtime Manager UI, the platform shows all the build tags for Edge and LTS channels. The build tag is specific to Runtime Manager and CloudHub 2.0 because the image can be built multiple times for a given Mule runtime version. 
```